<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dairy</class>
 <widget class="QMainWindow" name="Dairy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Дневник</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/img/icon_dairy5.png</normalon>
   </iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9 Декабря 2019</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>12</pointsize>
          <italic>false</italic>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit{
	background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                                stop: 0 #FAFBFE, stop: 1 #DCDEF1);
	border: 1px solid gray;
	border-radius: 3px;
}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="overwriteMode">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="1.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="1.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="1.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="1.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="1.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="1.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="1.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="1.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="1.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>250</red>
                <green>251</green>
                <blue>254</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>220</red>
                <green>222</green>
                <blue>241</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit{
	background: qlineargradient(x1: 1, y1: 0, x2: 0, y2: 0, stop: 0 #FAFBFE, stop: 1 #DCDEF1);
	border: 1px solid gray;
	border-radius: 3px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="undoRedoEnabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Выбор даты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Открыть календарь</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_day">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1752</year>
          <month>9</month>
          <day>14</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>d</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_month">
        <property name="displayFormat">
         <string>MMMM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_year">
        <property name="displayFormat">
         <string>yyyy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Открыть дневник</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Шрифт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fontSelect">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fontSize">
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Сохранить запись</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
